### DISCUSSION TITLE
Error when doing a hyperspecific thing

### DISCUSSION THREAD

**Discussion by soopax** [ppy/osu#23603] [Q&A] [answered]
Related to https://github.com/ppy/osu/discussions/23602.

### Steps to reproduce
1. Create a new multiplayer room
2. Start the match
3. In the loading screen, open the in-game beatmap listing (`Ctrl-B`)
4. Download a beatmap
5. When the beatmap has finished downloading, click on the arrow to go to the beatmap
6. Close the beatmap listing
7. Go back to the lobby

Log:
[runtime.log](https://github.com/ppy/osu/files/11523115/runtime.log)

**Comment by bdach** [comment, marked as answer]
Escalated to https://github.com/ppy/osu/issues/23611
